### ISSUE TITLE
Expand collection search to include full-text

### ISSUE THREAD

**Issue by o0beaner** [Manuel-777/MTG-Arena-Tool#1253]
In looking at the source, it does not look like the collection search looks at the actual body text on the card.

In the MTGA client itself, I could search for, say, "reveals their hand" to get every card within my filter that shows my my opponent's hand. I'm not sure what the card database schema looks like, but is there anything we can use to parse what the cards actually do and include that as searchable data?

**Comment by Manuel-777** [comment]
You are right, mtgatool does not search by oracle. I am currently not storing the card's oracle because its just too much data to keep on memory for little value, also the filtering would be a little slower (but I guess its not that much of a problem)

The metadata source code is in another repo if you are interested;
https://github.com/mtgatool/mtgatool-metadata
